packages/compass-settings/src/stores/settings.ts

```diff
@@ -173,21 +229,37 @@ export const changeFieldValue = <K extends keyof UserConfigurablePreferences>(
   };
 };
 
-export const openModal = (): SettingsThunkAction<Promise<void>> => {
+export const openModal = (
+  tab?: SettingsTabId
+): SettingsThunkAction<Promise<void>, OpenSettingsModalAction> => {
   return async (dispatch) => {
-    dispatch({ type: ActionTypes.OpenSettingsModal });
+    dispatch({ type: ActionTypes.OpenSettingsModal, tab });
```

Comment:
... and this the other thing, I guess, although I don't know much about what potential alternatives would look like – this is moving the "open tab" state to redux from the React component, and this will enable us to open a specific tab in the modal from other parts of Compass (which is planned for the next PR)

Reply:
Yeah, that's definitely the way to go about it. We can also now change the welcome modal to link directly to privacy tab if we want to

Reply:
Oh, yeah, that makes a ton of sense 🙂 Done in 20a2084c037a1c40e2e6b4ab9f92c01c814f222d!